nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/MSBuildHelperTests.cs

```diff
@@ -493,6 +497,792 @@ public async Task DependencyConflictsCanBeResolved()
         }
     }
 
+    // Scenario 1
```

Comment:
nit: not sure how common this is to be used, but would love to see all of these "Scenario X" tests wrapped in a [`#region`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#defining-regions)

Reply:
Looks like you did it, but could you name the region as well? Maybe like `NugetDependencyResolutionTests` or something?